lib/mp4/transmuxer.js

```diff
@@ -1082,7 +1080,9 @@ Transmuxer = function(options) {
   this.setBaseMediaDecodeTime = function(baseMediaDecodeTime) {
     var pipeline = this.transmuxPipeline_;
 
-    this.baseMediaDecodeTime = baseMediaDecodeTime;
+    if (!options.keepOriginalTimestamps) {
```

Comment:
Since this seems to change behavior, does it break https://github.com/videojs/mux.js/pull/329/files#diff-0f007c404c74edbc9d98f0a4d7f54f86R3533 ?

Reply:
I removed that comment. This does changes behavior but only very recent behavior, that I believe wasn't the real fix for what we needed.